README.md

```diff
@@ -93,17 +112,43 @@ need to do this only once:
 docker cp tomcat-users.xml scxa-tomcat:/usr/local/tomcat/conf
 ```
 
+Check your Tomcat logs and look for a line similar to the one below to make sure the new user settings have been
+loaded:
+```
+scxa-tomcat    | 12-Jan-2021 14:58:51.969 INFO [Catalina-utility-2] org.apache.catalina.users.MemoryUserDatabase.backgroundProcess Reloading memory user database [UserDatabase] from updated source [file:/usr/local/tomcat/conf/tomcat-users.xml]
+```
+
 Run the Gradle task `war` in the `atlas-web-single-cell` directory:
 ```bash
 cd atlas-web-single-cell
 ./gradlew -PbuildProfile=docker war
 ```
 
 You should now have the file `build/libs/gxa#sc.war` which by default Tomcat’s naming conventions will be served at
-`gxa/sc`. Point your browser at `http://localhost:8080/gxa/sc` and voilà!
+`gxa/sc`. Tomcat should automatically load the application after a few seconds. You should be seeing something like
+this in your logs:
+```
+scxa-tomcat    | 12-Jan-2021 14:59:47.566 INFO [Catalina-utility-1] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/gxa#sc.war] has finished in [5,510] ms
+```
+
+Point your browser at `http://localhost:8080/gxa/sc` and voilà!
 
 Every time you re-run the `war` task the web app will be automatically re-deployed by Tomcat.
 
+If you get any `war` redeploy issues or want to start again freshly, stop all the containers using this:
+
+```bash
+ATLAS_DATA_PATH=/path/to/sc/atlas/data \
+POSTGRES_HOST=scxa-postgres \
+POSTGRES_DB=gxpscxadev \
+POSTGRES_USER=atlasprd3 \
+POSTGRES_PASSWORD=atlasprd3 \
+docker-compose down
+```
+If you get any `java.net.UnknownHostException: Invalid hostname for server: local` exceptions while running application.
+Go to `profile-docker. gradle` file and check these attribute values `ext.zkHost` & `ext.solrHost` and compare with running container names.
+If they are different, update them.These should be ZooKeeper and Solr container names.
+
```

Comment:
This is something that was happening because my push contained wrong settings. I think this note is not necessary because the parameters are now fixed. Remember that this section is about getting things up/down smoothly. If you still want to keep this note add a `## Troubleshooting` section at the end.